### PR TITLE
Add breadcrumbs pattern

### PR DESCRIPTION
**Pull request by yunjungyeh**

### CODE REVIEW

File: site/docs/patterns/breadcrumbs.mdx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+### Anatomy
+
+Breadcrumbs contain:
+
+1. **Breadcrumb:** Previous levels displayed as links icons separating them.
+2. **Current level:** A text label showing the user’s current location.
+3. **SupOverflow menu:** Use an overflow menu to show collapsed breadcrumbs.  
+
+<Diagram
+  src="/img/patterns/breadcrumbs/breadcrumbs-anatomy.png"
```
Comment (JoanaMMoreira) [inline site/docs/patterns/breadcrumbs.mdx:47]:
```suggestion
  src="/img/patterns/breadcrumbs/breadcrumbs-anatomy-1.png"
```

File: site/docs/patterns/breadcrumbs.mdx [hunk 2]
```diff
@@ -0,0 +1,146 @@
+---
+title: Breadcrumbs
+layout: DetailPattern
+aliases:
+  - /salt/patterns/Breadcrumbs
```
Comment (joshwooding) [inline site/docs/patterns/breadcrumbs.mdx:5]:
```suggestion
  - /salt/patterns/breadcrumbs
```

File: site/docs/patterns/breadcrumbs.mdx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  alt="Example showing breadcrumbs."
+  border
+/>
+
+## When to use
+
+Use the pattern when the user navigates through related pages, or drills down into tables or charts. It provides further navigation relative to a users workflow/selection within their current area, page or widget. They are especially useful for identifying a location in a hierarchy when the user lands on a page via deep-linking from search or an external source. 
+
+- **Default action:** Paste.
+- **Supplementary actions:** Keep source formatting, Match destination formatting, Paste text only.
```
Comment (joshwooding) [inline site/docs/patterns/breadcrumbs.mdx:23]: These seem to be from split button?

File: site/docs/patterns/breadcrumbs.mdx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+## How to build
+
+### Anatomy
+
+Breadcrumbs contain:
+
+1. **Breadcrumb:** Previous levels displayed as links icons separating them.
+2. **Current level:** A text label showing the user’s current location.
+3. **SupOverflow menu:** Use an overflow menu to show collapsed breadcrumbs.
```
Comment (joshwooding) [inline site/docs/patterns/breadcrumbs.mdx:44]:
```suggestion
3. **Overflow menu:** Use an overflow menu to show collapsed breadcrumbs.  
```

**Review by joshwooding — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**